src/db/categories.js

```diff
@@ -0,0 +1,114 @@
+import geog from "../assets/geog.svg";
+import history from "../assets/history.svg";
+import math from "../assets/maths.svg";
+import book from "../assets/book.svg";
+import comics from "../assets/comics.svg";
+import science from "../assets/science.svg";
+import tech from "../assets/tech.svg";
+import sports from "../assets/sports.svg";
+import vehicle from "../assets/vehicle.svg";
```

Comment:
add the asset to the public folder and then access it

Reply:
ok will do that